src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/CreateTableOperation.java

```diff
@@ -55,9 +55,9 @@ public CreateTableOperation(
   @Override
   public Uni<Supplier<CommandResult>> execute(
       DataApiRequestInfo dataApiRequestInfo, QueryExecutor queryExecutor) {
-
-    CreateTableStart create =
-        createTable(commandContext.schemaObject().name.keyspace(), tableName).ifNotExists();
+    String keyspaceQuoted = "\"%s\"".formatted(commandContext.schemaObject().name.keyspace());
```

Comment:
Do we not have helper method(s) in `CqlIdentifier` to handle quoting? Instead of manually surrounding with double quotes.

Reply:
I did not find any method in driver that gives us CreateTableStart back and handle quote automatically.

```
    @NonNull
    public static CreateTableStart createTable(@NonNull String tableName) {
        return createTable(CqlIdentifier.fromCql(tableName));
    }

    @NonNull
    public static CreateTableStart createTable(@Nullable String keyspace, @NonNull String tableName) {
        return createTable(keyspace == null ? null : CqlIdentifier.fromCql(keyspace), CqlIdentifier.fromCql(tableName));
    }
```
I think they design this by the tableName/keyspace as cql representation. Which means if we want quoted string, we need to add by ourselves.

Reply:
That seems very odd -- there's nothing that takes `CqlIdentifier`? We should pass

   CqlIdentifier.fromInternal(tableName)

to ensure quoting is used if (and only if) necessary. But I agree, if only `String` is allowed then we must quote.